flash-messages/Gemfile

```diff
@@ -0,0 +1,20 @@
+source "https://rubygems.org"
+
+gem "sinatra"
+gem "data_mapper"
```

Comment:
Is a database necessary to illustrate flash? (I don't really know, it probably is helpful)